home/factories.py

```diff
@@ -121,6 +122,13 @@ class FooterIndexPageFactory(PageFactory):
     class Meta:
         model = FooterIndexPage
 
+
+class SurveyFactory(PageFactory):
+    title = factory.Sequence(lambda n: f'survey{n}')
+
+    class Meta:
+        model = Survey
```

Comment:
I think this should be in the questionnaire module. Dependencies should flow towards home, not the other way.

Reply:
Removed this factory from home module.